environment/metadata.go

```diff
@@ -136,6 +138,7 @@ func registerEKSData(d *MetaDataStrings) {
 func registerEKSE2ETestData(dataString *MetaDataStrings) {
 	flag.StringVar(&(dataString.Region), "region", "", "AWS region")
 	flag.StringVar(&(dataString.K8sVersion), "k8s_version", "", "Kubernetes version")
+	flag.StringVar(&(dataString.Destroy), "destroy", "false", "Whether to run in destroy mode (true/false)")
```

Comment:
Can use https://pkg.go.dev/flag#BoolVar instead of using a string and later converting it